docs/modules/SDDs/pages/0024-naming-scheme-v2.adoc

```diff
@@ -0,0 +1,76 @@
+= SDD 0024 - Project Syn Tools Naming Scheme v2
+
+:sdd_author:    Tobias Brunner
+:sdd_owner:
+:sdd_reviewers:
+:sdd_date:      2020-10-23
+:sdd_status:    accepted
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+Naming is hard, this page helps to make it easy. It describes the rules to name Project Syn tools.
+====
+
+== Motivation
+
+In xref:0002-naming-scheme.adoc[SDD 0002 - Naming Scheme] a naming scheme was defined for Project Syn tools which turned out to be sub-optimal.
+Having tools with non-descriptive names increases the (already relatively high) barrier of entry into the world of Project Syn.
+This SDD describes a more descriptive naming scheme which solves this issue.
+
+=== Goals
+
+* Naming scheme which is descriptive
+* Applicable to all new tools from now on
+
+=== Non-Goals
+
+* Renaming of existing tools (like Lieutenant, Steward and Commodore) as they will change over time anyways, ultimately being replaced by newer tools.
+
+== Design Proposal
+
+Tools must have descriptive display names of the form `Syn ${type}[ ${purpose}]`.
```

Comment:
I prefer to have the type at the end. It feels more natural when speaking.

```suggestion
Tools must have descriptive display names of the form `Syn[ ${purpose}] ${type}`.
```

Reply:
Maybe we could loosen this up even further and don't require a certain ordering of the purpose and type?

Reply:
No, not defining the order will results in both variants. This can again be confusing. You never now if tool use one or the other order. This makes it hard for the muscle memory.